src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationUtilities.cs

```diff
@@ -109,8 +111,10 @@ internal static class SemanticClassificationUtilities
                 {
                     var classifiedSpans = ClassificationUtilities.GetOrCreateClassifiedSpanList();
 
-                    await classificationService.AddSemanticClassificationsAsync(
-                        document, snapshotSpan.Span.ToTextSpan(), classifiedSpans, cancellationToken: cancellationToken).ConfigureAwait(false);
+                    var newSpans = await classificationService.GetSemanticClassificationsAsync(
+                        document, snapshotSpan.Span.ToTextSpan(), cancellationToken).ConfigureAwait(false);
+
+                    CombineSpans(classifiedSpans, newSpans);
```

Comment:
I'm confused about this.  classifiedSpans is just an empty list.  So what is being COmbinedHere.